### ISSUE TITLE
Truecaser Known Case Tokens

### ISSUE THREAD

**Issue by Patdue** [alvations/sacremoses#16]
If a word is not the first word of the sentence, and the word was seen with this exact casing in the training material, the original script does not recase the word.

i.e.
```bash
perl train - truecaser.perl --model big.model --corpus big.txt
```
```bash
echo "THE ADVENTURES OF SHERLOCK HOLMES" | perl truecase.perl --model big.model
the ADVENTURES OF SHERLOCK HOLMES
```

**Comment by alvations** [comment]
This I'm not sure why would that be expected, yes if we were to emulate that, it should keep the caps cases but it's strange that truecasing didn't catch that in Moses.

**Comment by alvations** [comment]
This should be patched by #39 

Thanks @Patdue for spotting the issue!
